packages/block/src/block/block.ts

```diff
@@ -158,7 +158,7 @@ export class Block {
    * Returns a Array of the raw Bytes Arrays of this block, in order.
    */
   raw(): BlockBytes {
-    const bytesArray = <BlockBytes>[
+    const bytesArray = [
```

Comment:
Isn't it more correct to
```suggestion
    const bytesArray: BlockBytes = [
```

(nit)

Reply:
Good catch.  Have corrected